### PR TITLE
Review backlinks, header, and forms. Entirely.

### PR DESCRIPTION
**Pull request by lukeocodes**

### CODE REVIEW

File: blog/src/components/nav/Backlink.astro [hunk 1]
```diff
@@ -4,4 +4,4 @@ import Link from "../../shared/components/general/Link.astro";
 const { href, class: classes } = Astro.props;
 ---
 
-<Link {href} class:list={["inline-block py-4 md:py-6 lg:py-8 xl:py-10", classes]} icon="arrow-left"><slot /></Link>
+<Link {href} class:list={["inline-block nudge-icon nudge-icon--left", classes]} icon="arrow-left"><slot /></Link>
```
Comment (SandraRodgers) [inline blog/src/components/nav/Backlink.astro:7]: The designs show backlink as 18px so this needs that style added

File: shared/src/shared/components/layout/Header.astro [hunk 2]
```diff
@@ -65,19 +66,17 @@ import MobileNavItem from "../nav/MobileNavItem.astro";
...
-						<Icon icon="search" class="w-[1.5em] h-[1.5em] fill-current" />
-					</button>
-					<NavItem href="https://console.deepgram.com/" rel="noopener noreferrer nofollow" class="button button--small text-white">Log In</NavItem>
-					<NavItem href="/contact" class="button button--secondary button--small">Contact Us</NavItem>
-					<NavItem href="https://console.deepgram.com/signup" rel="noopener noreferrer nofollow" class="ml-4 button button--primary button--small">Sign Up</NavItem>
+				<div class="flex space-x-[1rem] items-center xl:ml-12">
+					<Button icon="search" class="button big-icon button--main-nav" />
+					<Link href="https://console.deepgram.com/" class="button button--small button--main-nav text-white">Log In</Link>
+					<Link href="/contact" class="button button--secondary button--small w-[7rem]">Contact Us</Link>
+					<Link href="https://console.deepgram.com/signup" class="button button--primary button--small w-[7rem]">Sign Up</Link>
```
Comment (SandraRodgers) [inline shared/src/shared/components/layout/Header.astro:79]: The spacing on this doesn't look right on smaller screen sizes. 
Screen Shot 2022-08-29 at 12 52 43 PM

File: shared/src/shared/assets/styles/main.css [hunk 3]
```diff
@@ -102,22 +102,130 @@ code {
...
+
+	.big-icon svg {
+		@apply w-[1.4rem];
+	}
+
+	.astro-code {
+		@apply overflow-hidden py-7 px-10;
+	}
+
+	.draw-underline span {
```
Comment (SandraRodgers) [inline shared/src/shared/assets/styles/main.css:157]: Could you add these examples to the component library? It's easier to find examples of them that way

File: blog/src/components/nav/Backlink.astro (hunk 1 above)
Comment (SandraRodgers) [reply blog/src/components/nav/Backlink.astro:7]: Also needs semibold font-weight

File: shared/src/shared/components/layout/Header.astro (hunk 2 above)
Comment (lukeocodes) [reply shared/src/shared/components/layout/Header.astro:79]: > 

Ahh fixed in a later PR

File: blog/src/components/nav/Backlink.astro (hunk 1 above)
Comment (lukeocodes) [reply blog/src/components/nav/Backlink.astro]: Fixed! :)

**Review by SandraRodgers — APPROVED** [review]

**Pull request merged**